manifests/exporter.pp

```diff
@@ -19,21 +19,26 @@
     server_enabled => true
   }
 
+  $_rsan_clients = $rsanip.reduce('') |$memo, $ip| {
```

Comment:
Can we get a comment here on what it does. Reduce functions are very hard to get your head around for many people

Reply:
Absolutely. I will add a comment in there. If there is a pattern that is more common or easier to read, let me know and I can change it.